src/steamship/agents/llms/steamship_llm.py

```diff
@@ -0,0 +1,123 @@
+import logging
+from pprint import pformat
+from typing import List, Optional
+
+from steamship import Block, File, MimeTypes, PluginInstance, Steamship, SteamshipError, Tag
+from steamship.agents.schema import Tool
+from steamship.data import GenerationTag, TagKind
+from steamship.plugin.capabilities import (
+    CapabilityPluginRequest,
+    CapabilityPluginResponse,
+    ConversationSupport,
+    FunctionCallingSupport,
+    SystemPromptSupport,
+)
+
+
+class SteamshipLLM:
+    """A class wrapping LLM plugins."""
+
+    def __init__(self, plugin_instance: PluginInstance):
+        # TODO (PR callout): In the spirit of getting folks to pin to specific instances more often, take the route of
+        #  using specific PluginInstances rather than client.use_plugin all the time with the params that we determine?
+        self.client = plugin_instance.client
+        self.plugin_instance = plugin_instance
+
+    class Impls:
+        @staticmethod
+        def gpt(
```

Comment:
May want a `version_handle` arg for these to pass through to use_plugin, otherwise there's no way to control what version you get.

Reply:
`model` handles that for this specific plugin usage, but yes, e.g. Replicate would want that.

Reply:
I just realized this probably means version of the plugin, rather than version of the model.